webapp/src/components/AssetDetailPage/AssetDetailPage.container.js

```diff
@@ -1,25 +1,25 @@
 import { withRouter } from 'react-router'
 import { connect } from 'react-redux'
 
-import { getCenterCoords } from 'shared/asset'
 import { locations } from 'locations'
 import { navigateTo } from 'modules/location/actions'
-import { getError, isLoading } from 'modules/parcels/selectors'
-
 import AssetDetailPage from './AssetDetailPage'
+import { ASSET_TYPES } from '../../shared/asset'
+import { buildCoordinate } from '../../shared/parcel'
 
-const mapState = (state, { asset }) => {
-  const { x, y } = getCenterCoords(asset)
+const mapState = (state, { location, match }) => {
+  const assetType = Object.keys(ASSET_TYPES)
+    .map(key => ASSET_TYPES[key])
+    .find(assetType => location.pathname.includes(ASSET_TYPES[assetType]))
```

Comment:
Support for old parcel URLs breaks!

![image](https://user-images.githubusercontent.com/692077/46730911-9956f980-cc89-11e8-90e4-ad40e0c28197.png)

Reply:
fixed in https://github.com/decentraland/marketplace/pull/549/commits/ec4510dc29ac78b6c0cf4d602d697628033c5b39